web-admin/src/components/projects/ShareProjectCTA.svelte

```diff
@@ -1,3 +1,31 @@
-<div class="border bg-gray-50 p-2 text-gray-400">
-  Placeholder for sharing CTA
-</div>
+<script lang="ts">
+  import AccessControls from "@rilldata/web-admin/components/access-controls/AccessControls.svelte";
+  import CLICommandDisplay from "@rilldata/web-common/components/commands/CLICommandDisplay.svelte";
+
+  export let organization: string;
+  export let project: string;
+
+  let addUserCommand: string;
+  $: addUserCommand = `rill user add --org ${organization} --project ${project} --role viewer`;
+</script>
+
+<AccessControls {organization} {project}>
+  <svelte:fragment slot="manage-project">
+    <span class="uppercase text-gray-500 font-semibold text-[10px] leading-4"
+      >Share</span
+    >
+    <div>
+      Run this command in the Rill CLI to invite a teammate to view this
+      project.
+    </div>
+    <CLICommandDisplay command={addUserCommand} />
+  </svelte:fragment>
+  <svelte:fragment slot="read-project">
+    <span class="uppercase text-gray-500 font-semibold text-[10px] leading-4"
+      >Share</span
```

Comment:
We can put the `Share` title above the slots so that it's not repeated for both the cases.